src/Stratis.SmartContracts.CLR/ContractModuleDefinition.cs

```diff
@@ -17,12 +17,10 @@ public sealed class ContractModuleDefinition : IContractModuleDefinition
         private List<TypeDefinition> contractTypes;
 
         private TypeDefinition contractType;
-        private readonly MemoryStream stream;
 
-        public ContractModuleDefinition(ModuleDefinition moduleDefinition, MemoryStream stream)
+        public ContractModuleDefinition(ModuleDefinition moduleDefinition)
```

Comment:
We should still keep the stream around here as there's no guarantee that it gets disposed by cecil.